src/providers/Notifications.tsx

```diff
@@ -47,6 +50,37 @@ export const useNotifications = () => {
   return { showNotification, enqueueDefaultSnackbar, closeNotification };
 };
 
+export const useTxSuccessNotification = (
```

Comment:
consider moving it to some more gateway or transaction-related place. This file is for very generic notification stuff. Like gatewayHooks.ts or transactionsHooks.ts.

Reply:
done